<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Research | Ziqiao Wang </title> <meta name="author" content="Ziqiao Wang"> <meta name="description" content="Ziqiao Wang's Academic Website. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="Ziqiao Wang, JHU, Biostatistics, Johns Hopkins University, genetics, genomics"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon_zw.png?de402b36839d07092d3a75ffa5e3ab32"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://ziqiaow.github.io/research/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Ziqiao Wang </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">Home </a> </li> <li class="nav-item active"> <a class="nav-link" href="/research/">Research <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/software/">Software </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">CV </a> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Research</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <p>A full list of my publications are available here: <a href="https://scholar.google.com/citations?user=n6MtCyUAAAAJ&amp;hl=en&amp;oi=ao" target="_blank" rel="external nofollow noopener">Google Scholar</a> </p> <p> </p> <hr> <h5 id="efficient-modeling-of-polygenic-risk-scores-and-environment-interplay"><strong>Efficient Modeling of Polygenic Risk Scores and Environment Interplay</strong></h5> <p>We developed a series of novel methods for modeling of polygenic scores (PGS) in different epidemiological study designs. These include joint modeling of polygenic gene-environment correlations and interactions in case-control studies, and unified modeling of direct, indirect, and gxe interactions in family-based studies. We have shown improved efficiency in case-control studies that are comparable to cohort studies. Family-based methods can further eliminate potential population stratification bias in PGS estimations.</p> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/research_gxe-480.webp 480w,/assets/img/research_gxe-800.webp 800w,/assets/img/research_gxe-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/research_gxe.png" width="90%" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <ul> <li> <strong>Wang, Z.</strong>, Shi, W., Carroll, R., Chatterjee, N. (2024). Joint Modeling of Gene-Environment Correlations and Interactions Using Polygenic Risk Scores in Case-Control Studies. <em><strong>American Journal of Epidemiology</strong></em>. DOI: 10.1093/aje/kwae081 <a href="https://doi.org/10.1093/aje/kwae081" rel="external nofollow noopener" target="_blank">[Paper]</a> <a href="https://github.com/ziqiaow/RetroGE" rel="external nofollow noopener" target="_blank">[Software]</a> </li> <li> <strong>Wang, Z.</strong>, Ray, D., Grosvenor, L., Beaty, T., Volk, H., Ladd-Acosta, C., Chatterjee, N. (2024+). Novel Methods for Estimating Risk Parameters Associated with Polygenic Scores using Case-Parent Trio Designs. <a href="">[Paper]</a> <a href="https://github.com/ziqiaow/PRS-TRI" rel="external nofollow noopener" target="_blank">[Software]</a> </li> </ul> <p> </p> <hr> <h5 id="multi-omics-data-integration"><strong>Multi-Omics Data Integration</strong></h5> <p>We developed novel methods for data integrations to examine the associations between genes and disease phenotype through multiple layers of omic information. Our methods are based on multivariate Gaussian mixture models to characterize the precise correlations between different data types using summary statistics, leading to superior statistical power in association studies. We have also extended this work to spatially resolved high-dimensional genomics data, including DNA methylation and RNA sequencing data using specific datasets in bladder tumor samples, to increase the power of finding molecular signals contributing to cancer initiation, promotion, and progression.</p> <p>Our work has several key advantages over conventional designs, including the novel feature of controlling the false discovery rate in significant genes across multiple omics data types, the ability to incorporate independent or overlapping sets of cohorts with respect to omics data, the generality of applications to meta-analysis and gene enrichment pathways, and fast computational efficiency by adopting the expectation-maximization (EM) algorithm.</p> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/research_spatialimix-480.webp 480w,/assets/img/research_spatialimix-800.webp 800w,/assets/img/research_spatialimix-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/research_spatialimix.png" width="90%" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <ul> <li> <strong>Wang, Z.</strong>, Wei, P. (2020). IMIX: A Multivariate Mixture Model Approach to Association Analysis Through Multi-Omics Data Integration. <em><strong>Bioinformatics</strong></em>. DOI:10.1093/bioinformatics/btaa1001 <a href="https://doi.org/10.1093/bioinformatics/btaa1001" rel="external nofollow noopener" target="_blank">[Paper]</a> <a href="https://github.com/ziqiaow/IMIX" rel="external nofollow noopener" target="_blank">[Software]</a> <strong>(Platform presentation, ASHG 2020)</strong> </li> <li> <strong>Wang, Z.</strong>, Czerniak, B., Wei, P. (2023). Spatial IMIX: A Mixture Model Approach to Spatially Correlated Multi-Omics Data Integration. <em>Preprint</em>. DOI:10.1101/2023.07.15.549148 <a href="https://www.biorxiv.org/content/10.1101/2023.07.15.549148v1" rel="external nofollow noopener" target="_blank">[Paper]</a> <a href="https://github.com/ziqiaow/spatialimix" rel="external nofollow noopener" target="_blank">[Software]</a> </li> </ul> <p> </p> <hr> <h5 id="epigenome-wide-association-studies-in-cancer"><strong>Epigenome-Wide Association Studies in Cancer</strong></h5> <p>We also investigated the role of DNA methylation in whole blood affecting susceptibility to pancreatic cancer. We performed a two-phase epigenome-wide association study by examining two independent case-control studies measured within MDA and leveraged the Framingham Heart Study data to increase the sample size in controls. We discovered important differentially methylated CpG sites and regions associated with pancreatic cancer, and validated the results using a third independent dataset from the Women’s Health Initiative. We further confirmed the causal relationship of the discovered CpG sites using Mendelian randomization analysis, and investigated the functional consequences in gene expression. We performed another large-scale EWAS in studying the role of differentially methylated CpG sites between HPV-positive and HPV-negative oropharyngeal cancer patients and discovered a long non-coding RNA nc886 that leads to further important clinical implications.</p> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/research_methylation-480.webp 480w,/assets/img/research_methylation-800.webp 800w,/assets/img/research_methylation-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/research_methylation.jpg" width="80%" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <ul> <li> <strong>Wang, Z.</strong>, Lu, Y., Fornage, M., Jiao, L., Li, D., Wei, P. (2022). Identification of novel susceptibility methylation loci for pancreatic cancer in a two-phase epigenome-wide association study. <em><strong>Epigenetics</strong></em>. DOI: 10.1080/15592294.2022.2026591 <a href="https://doi.org/10.1080/15592294.2022.2026591" rel="external nofollow noopener" target="_blank">[Paper]</a> <strong>(Platform presentation, IGES 2019)</strong> </li> <li>Xu, Y.(co-first author), <strong>Wang, Z.</strong>(co-first author), Wei, P., Gairola, R., Kelsey, K., Sikora, A., Li, G., Gu, J. (2022). Hypermethylation of nc886 in HPV-positive oropharyngeal cancer and its clinical implications: an epigenome DNA methylation profiling study. <em><strong>Molecular Therapy-Nucleic Acids</strong></em>. DOI:10.1016/j.omtn.2022.11.012 <a href="https://doi.org/10.1016/j.omtn.2022.11.012" rel="external nofollow noopener" target="_blank">[Paper]</a> </li> </ul> <p> </p> <hr> <h5 id="method-for-benefit-risk-analysis-in-clinical-trials"><strong>Method for Benefit-Risk Analysis in Clinical Trials</strong></h5> <p>I collaborated with Dr. Jie Chen at Merck during my summer internship and developed a novel method to test for significant trend effects of multiple dosage levels of medication use by using composite endpoints in clinical trial studies. This method prioritizes the user’s intended endpoints including the adverse events and efficacy endpoints and assesses whether the examined dose levels in clinical trials show an increasing or decreasing trend effect. This method is computationally fast and robust to type I errors.</p> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/research_dose-480.webp 480w,/assets/img/research_dose-800.webp 800w,/assets/img/research_dose-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/research_dose.png" width="75%" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <ul> <li> <strong>Wang, Z.</strong>, Chen J. Testing for Trend in Benefit-Risk Analysis with Prioritized Multiple Outcomes. (2020). <em><strong>Statistics in Biopharmaceutical Research</strong></em>. DOI: 10.1080/19466315.2019.1690037 <a href="https://www.tandfonline.com/doi/full/10.1080/19466315.2019.1690037" rel="external nofollow noopener" target="_blank">[Paper]</a> <a href="https://github.com/ziqiaow/MvTrend" rel="external nofollow noopener" target="_blank">[Software]</a> </li> </ul> <p> </p> <hr> <h5 id="collaborations-in-biomedical-sciences"><strong>Collaborations in Biomedical Sciences</strong></h5> <p>I welcome collaborations with biologists, physicians, epidemiologists, and other biomedical researchers. I did extensive collaboration work with biomedical researchers at MD Anderson Cancer Center during my PhD study. Applying my knowledge in statistical genetics and genomics, I supported a variety of important research work in molecular biology studying bladder cancer, pancreatic cancer, and head and neck cancer.</p> <ul> <li>Bondaruk, J.(co-first author), Jaksik, R.(co-first author), <strong>Wang, Z.</strong>(co-first author), Cogdell, D.(co-first author), Lee, S., Chen, Y., Dinh, K., Majewski, T., Zhang, L., Cao, S., Yao, H., Weinstein, J., Navai, N., Dinney, C., Gao, J., Theodorescu, D., Logothetis, C., Guo, C., Wang, W., McConkey, D., Wei, P., Kimmel, M., Czerniak, B. (2022). The origin of bladder cancer from mucosal field effects. <em><strong>iScience</strong></em>. DOI: 10.1016/j.isci.2022.104551 <a href="https://www.cell.com/iscience/fulltext/S2589-0042(22)00823-9" rel="external nofollow noopener" target="_blank">[paper]</a> </li> <li>Yang, G.(co-first author), Bondaruk, J.(co-first author), Cogdell, D.(co-first author), <strong>Wang, Z.</strong>(co-first author), Lee, S., Lee, J., Zhang, S., Choi, W., Wang, Y., Liang, Y., Wang, G., Wang, Y., Yao, H., Dadhania, V., Logothetis, C., Siefker-Radtke, A., Kamat, A., Dinney, C., Theodorescu, D., Kimmel, M., Wei, P., Guo, C., Weinstein, J., McConkey, D., Czerniak, B. (2020). Urothelial-to-Neural Plasticity Drives Progression to Small Cell Bladder Cancer. <em><strong>iScience</strong></em>. DOI: 10.1016/j.isci.2020.101201. <a href="https://www.cell.com/iscience/fulltext/S2589-0042(20)30386-2" rel="external nofollow noopener" target="_blank">[paper]</a> </li> <li>Lee, S., Bondaruk, J., Wang, Y., Chen, H., Lee, J., Majewski, T., Mullen, R., Cogdell, D., Chen, J., <strong>Wang, Z.</strong>, Yao, H., Kus, P., Jeong, J., Lee, I., Choi, W., Navai, N., Guo, C., Dinney, C., Baggerly, K., Mendelsohn, C., McConkey, D., Behringer, R., Kimmel, K., Wei, P., Czerniak, B. Loss of LPAR6 and CAB39L dysregulates the basal-to-luminal urothelial differentiation program, contributing to bladder carcinogenesis. (2024). <em><strong>Cell reports</strong></em>. DOI: 10.1016/j.celrep.2024.114146. <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(24)00474-1" rel="external nofollow noopener" target="_blank">[paper]</a> </li> <li>Guo, C. C., Bondaruk, J., Yao, H., <strong>Wang, Z.</strong>, Zhang, L., Lee, S., Lee, J. G., Cogdell, D., Zhang, M., Yang, G., Dadhania, V., Choi, W., Wei, P., Gao, J., Theodorescu, D., Logothetis, C., Dinney, C., Kimmel, M., Weinstein, J. N., McConkey, D. J., … Czerniak, B. (2020). Assessment of Luminal and Basal Phenotypes in Bladder Cancer. <em><strong>Scientific reports</strong></em>. DOI: 10.1038/s41598-020-66747-7. <a href="https://www.nature.com/articles/s41598-020-66747-7" rel="external nofollow noopener" target="_blank">[paper]</a> </li> <li>Guo, C., Majewski, T., Zhang, L., Yao, H., Bondaruk, J., Wang, Y., Zhang, S., <strong>Wang, Z.</strong>, Lee, J., Lee, S., Cogdell, D., Zhang, M., Wei, P., Grossman, H., Kamat, A., Duplisea, J., Ferguson, J., Huang, H., Dadhania, V., Dinney, C., Weinstein, J., Baggerly, K., McConkey, D., Czerniak, B. (2019). Dysregulation of EMT Drives the Progression to Clinically Aggressive Sarcomatoid Bladder Cancer. <em><strong>Cell reports</strong></em>. DOI: 10.1016/j.celrep.2019.04.048. <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(19)30521-2" rel="external nofollow noopener" target="_blank">[paper]</a> </li> <li>Zhang, Y., Sturgis, E.M., Wei, P., Liu, H., <strong>Wang, Z.</strong>, Ma, Y., Liu, C., Gu, K.J., Wei, Q., and Li, G. A Genetic Variant within MDM4 3’UTR miRNA Binding Site is Associated with HPV16-positive Tumors and Survival of Oropharyngeal Cancer. (2019). <em><strong>Molecular carcinogenesis</strong></em>. DOI:10.1002/mc.23116. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/mc.23116" rel="external nofollow noopener" target="_blank">[paper]</a> </li> <li>Yang, Z., Zhang, Y., Wang, X., Huang, J., Guo, W., Wei, P., Li, G., <strong>Wang, Z.</strong>, Huang, Z., and Zhang, L. Putative Biomarkers of Malignant Transformation of Sinonasal Inverted Papilloma into Squamous Cell Carcinoma. (2019). <em><strong>The Journal of international medical research</strong></em>. DOI:10.1177/0300060519838385. <a href="https://journals.sagepub.com/doi/full/10.1177/0300060519838385" rel="external nofollow noopener" target="_blank">[paper]</a> </li> </ul> <p> </p> <hr> <p>Currently at Johns Hopkins University, I am leading the effort in identifying important plasma proteins associated with common solid cancer incidence. I also performed Mendelian randomization analyses to identify potential causal relationships between proteins and incident hypertension. I have also supported projects constructing polygenic risk scores for type 2 diabetes and coronary artery disease, as well as unsupervised clustering using proteomics data to identify disease subtypes in stroke.</p> <ul> <li> <strong>Wang, Z.</strong>, Burk, V., Chatterjee, N., Platz, E. (2024+). Discovering plasma proteins associated with common solid cancer incidence in ARIC.</li> <li>Mathews, L., Hu, X., <strong>Wang, Z.</strong>, et al. (2024+). MMP7 as a Causal Factor for Incident Hypertension and its Complications Proteomic Analysis from the Atherosclerosis Risk in Communities (ARIC) Study and Mendelian Randomization Study.</li> <li>Dzaye, O., …, <strong>Wang, Z.</strong>, et al. (2024+). Polygenic Risk Scores and Extreme Coronary Artery Calcium Phenotypes in Adults &gt;75 Years Old: The Atherosclerosis Risk in Communities Study.</li> </ul> <p> </p> <hr> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Ziqiao Wang. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> </body> </html>